src/util/ReactUtils.ts

```diff
@@ -305,10 +305,17 @@ export const filterProps = (
    */
   const matchingElementTypeKeys = FilteredElementKeyMap?.[svgElementType] ?? [];
 
+  /**
+   * Props are blindly spread onto SVG elements. This loop filters out properties that we don't want to spread.
+   * Items filtered out are as follows:
+   *   - functions in properties that are SVG attributes
```

Comment:
```suggestion
   *   - functions
```

Reply:
All functions are filtered out.

Reply:
if `includeEvents` is true then there are functions that are passed down. `includeEvents` is only set to true when props are spread via our components (such as Curve) and never when props are spread onto an SVG element